### DISCUSSION TITLE
Thermal Runaway More Frequent After Last Update?

### DISCUSSION THREAD

**Discussion by mediamanrit** [Jyers/Marlin#848] [Q&A] [answered]
I was having somewhat frequent errors when running the bed leveling and it not finishing, so I tried updating to the latest firmware.  The bed leveling quirkyness is fixed, but now I get thermal runaway shutdowns on about 1/3 of my prints.  Same gcode files as before.

Is this just me, or are others seeing this as well?

**Comment by Jyers** [comment, marked as answer]
I did tweak thermal runaway settings in the last release. I have pulled them back a touch after a good few error reports. I'm trying to get it as safe as possible without without causing too many errors. It will hopefully be better in the next release.

**Reply by mediamanrit** [reply]
Thanks!

**Comment by ubeyou** [comment]
Same here, keep getitng thermal runaway

![WhatsApp Image 2021-05-17 at 10 35 29 AM](https://user-images.githubusercontent.com/935938/118427988-087a1180-b701-11eb-823d-c4c150ecb141.jpeg)

**Reply by Hammy1989** [reply]
I'm having the same thing occur
